pyomeca/processing/markers.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import xarray as xr
+
+
+def markers_from_rototrans(markers: xr.DataArray, rt: xr.DataArray) -> xr.DataArray:
```

Comment:
That names sounds weird to me... It is not exactly "from" more than applied/computed... but I get that this name standardize... It just bugged me up to now each time I have seen it

Reply:
Fair point...